docs/about/compatibility/data-type-support.rst

```diff
@@ -0,0 +1,547 @@
+      - HIP type
+      - Description
+    * 
+      - int8
+      - ``int8_t``, ``uint8_t``
+      - An signed or unsigned 8-bit integer.
+    * 
+      - int16
+      - ``int16_t``, ``uint16_t``
+      - An signed or unsigned 16-bit integer.
+    * 
+      - int32
+      - ``int32_t``, ``uint32_t``
+      - An signed or unsigned 32-bit integer.
+    * 
+      - int64
+      - ``int64_t``, ``uint64_t``
+      - An signed or unsigned 64-bit integer.
+
+Floating-point Types
+==========================================
+
+.. image:: ../../data/about/compatibility/floating-point-data-types.png
+    :alt: Supported floating-point types
+
+.. list-table::
+    :header-rows: 1
+    :widths: 15,15,70
+
+    * 
+      - Type name
+      - HIP type
+      - Description
+    *
+      - float8 (E4M3)
+      - ``-``
+      - A 8-bit floating-point number mostly following IEEE-754 conventions and bit layout **S1E4M3** described in `8-bit Numerical Formats for Deep Neural Networks <https://arxiv.org/abs/2206.02915>`_ , with expanded range and with no infinity or signed zero. NaN is represented as negative zero.
+    *
+      - float8 (E5M2)
+      - ``-``
+      - A 8-bit floating-point number mostly following IEEE-754 conventions and bit layout **S1E5M2** described in `8-bit Numerical Formats for Deep Neural Networks <https://arxiv.org/abs/2206.02915>`_ , with expanded range and with no infinity or signed zero. NaN is represented as negative zero.
+    *
+      - float16
+      - ``half``
+      - A 16-bit floating-point number, conforming to the IEEE 754-2008 half precision storage format.
+    *
+      - bfloat16
+      - ``bfloat16``
+      - A shortened 16-bit version of the IEEE 754 single precision storage format.
+    *
+      - tensorfloat32
+      - ``-``
+      - Floating-point number that occupies 32 bits or less of storage, providing improved range compared to half (16-bit) format, at (potentially) greater throughput than single precision (32-bit) formats.
+    *
+      - float32
+      - ``float``
+      - A 32-bit floating-point number, conforming to the IEEE 754 single precision storage format.
+    *
+      - float64
+      - ``double``
+      - A 64-bit floating-point number, conforming to the IEEE 754 double precision storage format.
+
+.. note::
+
+  * The float8 and tensorfloat32 types are internal types used in calculations in the Matrix Cores and can be stored in any type of the same size.
+  * The encodings for FP8 (E5M2) and FP8 (E4M3) natively supported by MI300 differ from the FP8 (E5M2) and FP8 (E4M3) encodings used in H100 (`FP8 formats for Deep Learning <https://arxiv.org/abs/2209.05433>`_ ).
+  * In some AMD documents and articles float8 (E5M2) is called bfloat8.
+
+The support icon description
+==========================================
+
+.. list-table::
+    :header-rows: 1
+
+    * 
+      - The support icon explanation
+      - Support icon
+    * 
+      - Not supported
+      - ❌
+    * 
+      - Partial support
```

Comment:
this doesn't seem to be used in the document and we decided not to have this earlier. if nothing changed we should remove this

Reply:
We found one place, where it was needed.